internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -220,10 +220,14 @@ func (c *ClusterManager) processAcceptedClusters() []error {
 	for i := range acceptedClusters {
 		cluster := acceptedClusters[i]
 		glog.V(10).Infof("accepted cluster ClusterID = %s", cluster.ClusterID)
-		metrics.UpdateClusterStatusSinceCreatedMetric(cluster, api.ClusterAccepted)
-		if err := c.reconcileAcceptedCluster(&cluster); err != nil {
-			errs = append(errs, errors.Wrapf(err, "failed to reconcile accepted cluster %s", cluster.ID))
-			continue
+		if cluster.ClusterType != api.Enterprise.String() {
+			metrics.UpdateClusterStatusSinceCreatedMetric(cluster, api.ClusterAccepted)
+			if err := c.reconcileAcceptedCluster(&cluster); err != nil {
+				errs = append(errs, errors.Wrapf(err, "failed to reconcile accepted cluster %s", cluster.ID))
+				continue
+			}
+		} else {
+			glog.V(10).Infof("skipping reconciliation of accepted %s cluster with ClusterID = %s", api.Enterprise.String(), cluster.ClusterID)
```

Comment:
Thanks for adding this skip logic:

For now what we are missing is a database update to move the status of the cluster to the next phase 
e.g when the enterprise cluster is in accepted state, the only thing we'll do is move it to the cluster_provisioning state.

And in the method below
```go
processProvisioningClusters,
```

when the enterprise cluster is in provisioning state, the only thing we'll do is move it to `cluster_provisioned` state.

Lastly, in this method  
```go
processProvisionedClusters
```

the enterprise cluster, won't be terraformed but moved to the waiting_for_kas_fleetshard_operator state.